src/runtime/mini_webgpu.h

```diff
@@ -33,7 +33,7 @@
 #ifndef WEBGPU_H_
 #define WEBGPU_H_
 
```

Comment:
Did we put the clang-format comments in the `*_mini.h` files? Or were they there when we vendored them? Actually... did we vendor all of them?